### PR TITLE
[6] Develop a Product Page using HTML

### PR DESCRIPTION
**Pull request by michaelolubajo**
## Task: `HTML`
- `created and added productpage.html`

Screenshot_2020-07-18 Mi Band 5 Product Page

### CODE REVIEW

File: submissions/michaelolubajo/introductionToHtml/productpage.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+	<div class="topnav">
+  		<a class="active" href="#home">Home</a>
+  		<a href="#phones">Phones</a>
+ 		<a href="#mobilephonesaccessories">Mobile Phone Accessories</a>
+  		<a href="#smartwatches">Smart Watches</a>
+  		<a href="#miband5">Mi Band 5</a>
+	</div>
+	<hr>
+	<h2>Mi Band 5</h1>
+	<img src="images/1-mi-band-5.jpg" height="400" width="400">
```
Comment (ToluAkin) [inline submissions/michaelolubajo/introductionToHtml/productpage.html:21]: Remove the height and width in the images

File: submissions/michaelolubajo/introductionToHtml/productpage.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/michaelolubajo/introductionToHtml/productpage.html:1]: Well done!

File: submissions/michaelolubajo/introductionToHtml/productpage.html (hunk 1 above)
Comment (michaelolubajo) [reply submissions/michaelolubajo/introductionToHtml/productpage.html]: Done.

Comment (michaelolubajo) [reply submissions/michaelolubajo/introductionToHtml/productpage.html]: Screenshot_2020-07-20 Mi Band 5 Product Page

File: submissions/michaelolubajo/introductionToHtml/productpage.html (hunk 2 above)
Comment (michaelolubajo) [reply submissions/michaelolubajo/introductionToHtml/productpage.html:1]: Thanks. Please review latest push

**Pull request merged**